src/Stripe.net/Entities/Terminal/ConnectionTokens/ConnectionToken.cs

```diff
@@ -10,6 +10,9 @@ public class ConnectionToken : StripeEntity, IHasObject
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        [JsonProperty("location")]
+        public string Location { get; set; }
```

Comment:
This is an ID, correct? Should we rename it to `LocationId`?

Reply:
ah yes you're right. I forgot because it's not expandable and likely won't be, but consistency is important!